lib/src/rules/avoid_positional_boolean_parameters.dart

```diff
@@ -102,4 +113,17 @@ class _Visitor extends SimpleAstVisitor<void> {
       DartTypeUtilities.implementsInterface(
           node.identifier.bestType, 'bool', 'dart.core') &&
       !node.isNamed;
+
+  bool _isOverridingMember(Element member) {
```

Comment:
This jazz is largely copied from `lib/src/rules/annotate_overrides.dart`. I'd extract it to be a static utility, but it uses `manager`, instantiated in `visitCompilationUnit`. I wonder if this `manager` might become useful enough to create a parent visitor. `abstract class InheritanceAwareAstVisitor extends AstVisitor` or something...

Reply:
@bwilkerson: thoughts on that?

Reply:
We haven't decided yet whether InheritanceManager will be part of the public API or whether we'll have a different mechanism for answering questions like this. I'd postpone linter changes until the analyzer API is finalized.

Reply:
SG